README.md

```diff
@@ -18,27 +18,27 @@ See [IPFS getting-started](http://ipfs.io/docs/getting-started). If anything goe
 
 This module downloads `rust-ipfs` binaries from https://github.com/ipfs-rust/rust-ipfs/releases into your project.
 
-By default it will download the go-ipfs version that matches the npm version of this module. So depending on `rust-ipfs-dep@0.1.0` will install `rust-ipfs v0.1.0` for your current system architecture, in to your project at `node_modules/rust-ipfs-dep/ipfs`.
+By default it will download the rust-ipfs version that matches the npm version of this module. So depending on `rust-ipfs-dep@0.1.0` will install `rust-ipfs v0.1.0` for your current system architecture, in to your project at `node_modules/rust-ipfs-dep/ipfs`.
 
```

Comment:
I was thinking that maybe this README could have much less usage examples and emphasized text regarding the purpose (to run tests, measure progress, pre-alpha and all). Unsure of how much of the original we need to save, once published people might see this as an `go-ipfs` related alternative then wondering what's going on when it doesn't support the same amount of operations.

Reply:
Updated the README to add a Project Status: `Pre-Alpha` at the top. Hopefully that will deter people from getting too excited about it